etcdctl/ctlv2/command/backup_command.go

```diff
@@ -65,7 +65,7 @@ func handleBackup(c *cli.Context) {
 		destWAL = path.Join(c.String("backup-dir"), "member", "wal")
 	}
 
-	if err := os.MkdirAll(destSnap, 0700); err != nil {
+	if err := fileutil.TouchDirAll(destSnap); err != nil {
```

Comment:
we should still use mkdirall when we want to ensure we create a dir. touchdir will not return an error if the dir already exists.